src/flows/context/flow.query.tsx

```diff
@@ -278,6 +280,58 @@ export const FlowQueryProvider: FC = ({children}) => {
     status = RemoteDataState.Loading
   }
 
+  const queryDependents = (startID: string) => {
+    sendEvent(notebookQueryKey)
+    _queryDependents(startID)
+  }
+
+  const _queryDependents = (startID: string) => {
+    if (status === RemoteDataState.Loading) {
+      return
+    }
+
+    let map = generateMap(runMode === RunMode.Run)
```

Comment:
just curious why we're going with the run implementation vs preview. I no longer understand the difference or what the consequences of this condition is vs the alternative

Reply:
That makes two of us